tests_oval_graph/test_commands/test_command_line_client/test_client_arf_to_html.py

```diff
@@ -0,0 +1,76 @@
+import tempfile
+import uuid
+from pathlib import Path
+
+import pytest
+
+from oval_graph.command_line_client.arf_to_html import ArfToHtml
+
+from ...test_tools import TestTools
+from .global_test_feed import (EXPECTED_RULES_ID, GET_QUESTION_WITH_ARGS,
+                               SEARCH_BAD_RULES, SEARCH_RULES)
+
+PATH_TO_REPORT = Path('../../global_test_data/ssg-fedora-ds-arf.xml')
+TOP_PATH = Path(__file__).parent
+
+
+def get_client_arf_to_html(rule, optional_args=None):
+    patch = str(TOP_PATH / PATH_TO_REPORT)
+    args = ["--display", patch, rule]
+    if optional_args is not None:
+        args.extend(optional_args)
+    client = ArfToHtml(args)
+    client.load_file()
+    return client
+
+
+@pytest.mark.parametrize("rule, error_pattern", SEARCH_BAD_RULES)
+def test_expection_prepare_data(rule, error_pattern):
+    client = get_client_arf_to_html(rule)
+    rules = {'rules': [rule]}
+    with pytest.raises(Exception, match=error_pattern):
+        assert client.prepare_data(rules)
+
+
+@pytest.mark.parametrize("rule, error_pattern", SEARCH_BAD_RULES)
+def test_expection_search_rules_id(rule, error_pattern):
+    client = get_client_arf_to_html(rule)
+    with pytest.raises(Exception, match=error_pattern):
+        assert client.search_rules_id()
```

Comment:
Could we have a test that tests the `ValueError` and `NotTestedRule` exceptions raised at `oval_graph/command_line_client/client_arf_input.py` lines 56 or 58 ?